rootfs-builder/centos/config.sh

```diff
@@ -1,13 +1,48 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
 # This is a configuration file add extra variables to
 # be used by build_rootfs() from rootfs_lib.sh the variables will be
 # loaded just before call the function.
 
 # Here there are a couple of variables you may need.
 # Remove them or add more 
 
+# CentOS OS identifier (name)
+OS_NAME="Centos"
+
 # Centos Version
 OS_VERSION=${OS_VERSION:-7}
 
+# CentOS architecture
+ARCH=${ARCH:-"x86_64"}
```

Comment:
How about defining `ARCH` in `rootfs.sh` exactly as above so that this variable can be dropped from all the distro-specific `config.sh` files? If a particular distro needs to redefine that variable, it of course can do so.